user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.md

```diff
@@ -35,4 +35,4 @@ To do so:
 > Regenerating primary or secondary keys can also be done from the side panel.
 
 > [!NOTE]
-> Audit information about DCP keys is available on the *Organization* > *Audit* page.
+> Audit information about DCP keys is available on the *Organization* > *Audit* page (see [Audit logs in the cloud](xref:Auditing)).
```

Comment:
maybe DCP Audit logs or Audit logs in/on DCP to not create confusion about seeing all (dataminer) audit logs in the cloud?